test/e2e/push_test.go

```diff
@@ -76,6 +76,11 @@ var _ = Describe("Podman push", func() {
 		push := podmanTest.PodmanNoCache([]string{"push", "--tls-verify=false", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
 		push.WaitWithDefaultTimeout()
 		Expect(push.ExitCode()).To(Equal(0))
+
+		// Test --digestfile option
+		push2 := podmanTest.PodmanNoCache([]string{"push", "--tls-verify=false", "--digestfile=/tmp/digestfile.txt", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
+		push2.WaitWithDefaultTimeout()
+		Expect(push2.ExitCode()).To(Equal(0))
```

Comment:
@edsantiago Is there a way to check for the existence of the file here?

Reply:
@TomSweeneyRedHat I don't know: ginkgo and I do not get along. I tried some web searching and quickly got nowhere. I would imagine it's a combination of `os.Stat()` plus some form of `Expect()`, but really don't know. Sorry.

Reply:
@edsantiago no worries, I ran into the same thing last night and was hoping you'd a quick off the cuff answer.

Reply:
Why not os.Stat?

Reply:
os.stat should work, I was just thinking there'd be a more gingkoy way to do that.

Reply:
FYI @edsantiago, I kinda merged ginkgo and os.stat together.  I think it actually works out pretty well.